x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -282,6 +282,28 @@ str1:keyword |str2:keyword |ip1:ip  |ip2:ip
 // end::to_ip-result[]
 ;
 
+cdirMatchOrs
+required_capability: combine_disjunctive_cidrmatches
+
+FROM hosts 
+| WHERE CIDR_MATCH(ip1, "127.0.0.2/32") or CIDR_MATCH(ip1, "127.0.0.3/32") or CIDR_MATCH(ip0, "127.0.0.1") or CIDR_MATCH(ip0, "fe80::cae2:65ff:fece:feb9")
```

Comment:
Same here, you could alternate ip0 and ip1. And maybe some other CIDR_MATCH-free, unrelated predicate.

Reply:
Possibly another test with smth like `| WHERE CIDR_MATCH(ip0, "127.0.0.2") OR CIDR_MATCH(ip0, "127.0.0.3") AND CIDR_MATCH(ip1, "fe80::cae2:65ff:fece:fec0") | STATS c = COUNT(*)` (to get a 0, as predicates shouldn't be combined and match).

Reply:
More tests are added to CsvTests and QueryTranslatorTests.